crystal_functions/plot.py

```diff
@@ -5,6 +5,9 @@
 """
 
 
+from tkinter import Y
```

Comment:
Is this needed?

Reply:
@eascrizzi I resolved the conflict. It was just a different import, but I don't think either of them is needed.

Reply:
No it is not needed. Thank you very much!